packages/manager/modules/iam/src/components/createPolicy/createPolicy.controller.js

```diff
@@ -389,6 +389,9 @@ export default class CreatePolicyController {
         allow: this.model.actions.isWildcardActive
           ? [{ action: '*' }]
           : this.model.actions.selection
+              // actions tagged as embedded are covered by custom actions
+              // there is no need to persist them
+              .filter(({ embedded }) => !embedded)
```

Comment:
**nitpick(perf):** There is a `filter` running on line 396. Re-use the existing filter to avoid multiple looping .

Reply:
The second `filter` is only used to remove duplicated actions but at this step, the property `embedded` doesn't exist anymore. Indeed, the array is reduced to object as `[{action: 'actionValue'}]` therefore I can't use that `filter`